### ISSUE TITLE
softmax

### ISSUE THREAD

**Issue by hellosaferide** [131250208/TPlinker-joint-extraction#10]
在论文中你使用了softmax对标签进行对分类，然而在代码中，我没发现softmax函数，而是直接使用linear直接输出两个值

**Comment by 131250208** [comment]
@hellosaferide 你好，感谢关注~ 我们代码里的loss function使用的是[torch.nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=crossentropyloss#torch.nn.CrossEntropyLoss)，本身包含了softmax操作，所以不需要再自己做softmax了，可以参考官方文档。

**Comment by hellosaferide** [comment]
谢谢回答
